scipy/stats/_continuous_distns.py

```diff
@@ -9972,11 +9972,71 @@ class vonmises_gen(rv_continuous):
     `vonmises_line` is the same distribution, defined on :math:`[-\pi, \pi]`
     on the real line. This is a regular (i.e. non-circular) distribution.
 
-    `vonmises` and `vonmises_line` take ``kappa`` as a shape parameter.
+    Note about distribution parameters: `vonmises` and `vonmises_line` take
+    ``kappa`` as a shape parameter and the ``scale`` parameter does not have
+    any effect.
```

Comment:
I don't know why `vonmises_line` was documented here. I don't think it's necessary, personally, so while we're here, it's fine with me if you'd like to omit it.

If we're mentioning `scale` and removing the standard note:
![image](https://user-images.githubusercontent.com/6570539/235406714-9ef112c6-c342-4904-88f5-309eb78e4e75.png)

then we should mention `loc`, too.
```suggestion
    Note about distribution parameters: `vonmises` and `vonmises_line` take
    ``kappa`` as a shape parameter (concentration) and ``loc`` as the location
    (circular mean). A ``scale`` parameter is accepted but does not have any
    effect.
```

You might also want to state it as in the standard note: `vonmises.pdf(x, kappa, loc, scale)` is identically equivalent to `vonmises.pdf(x - loc, kappa)`.